test/language/test/language_test.gleam

```diff
@@ -1060,20 +1060,97 @@ fn bit_string_tests() -> List(Test) {
 }
 
 @target(erlang)
-fn bit_string_float_erlang() -> List(Test) {
+fn bit_string_target_tests() -> List(Test) {
   [
+    "<<0,1>> == <<1:size(16)>>"
+    |> example(fn() { assert_equal(True, <<0, 1>> == <<1:size(16)>>) }),
+    "<<0,0,0,1>> == <<1:32>>"
+    |> example(fn() { assert_equal(True, <<0, 0, 0, 1>> == <<1:32>>) }),
     "<<60,0>> == <<1.0:float-size(16)>>"
     |> example(fn() { assert_equal(True, <<60, 0>> == <<1.0:float-16>>) }),
     "<<63,128,0,0>> == <<1.0:float-32>>"
     |> example(fn() {
       assert_equal(True, <<63, 128, 0, 0>> == <<1.0:float-32>>)
     }),
+    "<<71, 108, 101, 97, 109>> == <<\"Gleam\":utf8>>"
+    |> example(fn() {
+      assert_equal(True, <<71, 108, 101, 97, 109>> == <<"Gleam":utf8>>)
+    }),
+    "bit_string from function"
+    |> example(fn() {
+      assert_equal(
+        True,
+        <<
+          0x1,
+          2,
+          2:size(16),
+          0x4:size(32),
+          "Gleam":utf8,
+          4.2:float,
+          <<<<1, 2, 3>>:bit_string, "Gleam":utf8, 1024>>:bit_string,
+        >> == importable.get_bit_string(),
+      )
+    }),
+    "bit_string module const"
+    |> example(fn() {
+      assert_equal(
+        True,
+        <<
+          0x1,
+          2,
+          2:size(16),
+          0x4:size(32),
+          "Gleam":utf8,
+          4.2:float,
+          <<<<1, 2, 3>>:bit_string, "Gleam":utf8, 1024>>:bit_string,
+        >> == importable.data,
+      )
+    }),
```

Comment:
Could you remove the duplicated tests out of the target-specific tests please 🙏 

Only tests that cannot be performed on the other targets should be in here.

Reply:
I removed them, but now I'm confused. This PR is for the js target only, if there shouldn't be any js target specific tests for this, what was I supposed to do in the first place when you asked me to add more tests there?

Reply:
I removed them but now I'm confused. This PR is for the js target only, so if there isn't supposed to be any js specific test in that file, what was I supposed to do there when you asked me to add more tests?